src/main/kotlin/no/nav/familie/ef/sak/ekstern/stønadsperiode/EksternStønadsperioderService.kt

```diff
@@ -24,6 +27,27 @@ class EksternStønadsperioderService(
         return EksternePerioderResponse(perioderFraITogEF)
     }
 
+    fun hentPerioderForOvergangsstønadOgSkolepenger(request: EksternePerioderRequest): OvergangsstønadOgSkolepengerResponse {
+        val perioderOS = periodeService.hentPerioderForOvergangsstønadFraEfOgInfotrygd(request.personIdent)
+        val begrensetPeriode = Datoperiode(fom = request.fomDato ?: LocalDate.MIN, tom = request.tomDato ?: LocalDate.MAX)
+
+        val eksternPeriodeMedStønadstypeOS =
+            perioderOS
+                .filter { periode ->
+                    Datoperiode(fom = periode.stønadFom, tom = periode.stønadTom).overlapper(begrensetPeriode)
+                }.map {
+                    EksternPeriodeMedStønadstype(
+                        it.stønadFom,
+                        it.stønadTom,
+                        StønadType.OVERGANGSSTØNAD,
+                    )
+                }
+
+        val eksternPeriodeMedStønadstypeSP = periodeService.hentPeriodeFraVedtakForSkolepenger(request.personIdent)
+
+        return OvergangsstønadOgSkolepengerResponse(request.personIdent, eksternPeriodeMedStønadstypeOS + eksternPeriodeMedStønadstypeSP)
```

Comment:
Hadde kanskje vært fint at responsen hadde 2 ulike lister for perioder av OS og SP? 
Eller slått de sammen @eirarset (?)

Reply:
Jeg fikser det slik dere ønsker, bare si ifra :)

Reply:
En liste er fint så lenge vi får typen for 🚀